app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Is this file (and other files like scaffolds.scss) used at all? It's not included in the CSS asset manifest file (application.css)